backend/oasst_backend/models/post.py

```diff
@@ -31,3 +31,5 @@ class Post(SQLModel, table=True):
     )
     payload_type: str = Field(nullable=False, max_length=200)
     payload: PayloadContainer = Field(sa_column=sa.Column(payload_column_type(PayloadContainer), nullable=True))
+    depth: int = Field(sa_column=sa.Column(sa.Integer, default=0, server_default=sa.text("0"), nullable=False))
```

Comment:
do we really need all these attributes?
it seems like sqlmodel might do the right thing with `depth: int = 0` without any field definition at all, no?

Reply:
There was an issue adding default boolean value (ack, done), the value did not propagate properly to existing rows. Solution was to specify server_default, so I did it the same way to other columns.